### PR TITLE
ready everything for github packages

### PR DESCRIPTION
**Pull request by oakfang**
Setup for a github package (see https://dev.to/dalenguyen/create-your-first-github-package-564f)

### CODE REVIEW

File: test/live-connection.ts [hunk 1]
```diff
@@ -75,17 +80,19 @@ test('End to end', done => {
...
         client1.subscribe(client2.jid);
     });
 
     client2.on('subscribe', () => {
+        console.log('c2 subscribe');
         client2.acceptSubscription(client1.jid);
     });
 
     client2.on('message', msg => {
+        console.log('c2 message');
```
Comment (paweldomas) [inline test/live-connection.ts:94]: are the log messages necessary?

Comment (oakfang) [reply test/live-connection.ts:94]: the tests failed without them 🤢

Comment (paweldomas) [reply test/live-connection.ts:94]: hmm that's super weird, it should work regardless, they pass for me with the logs removed

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -1,24 +0,0 @@
-name: Tests
```
Comment (paweldomas) [inline .github/workflows/ci.yaml:1]: why is this file removed? the way I understand it runs the tests via github actions, don't we need that?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -1,20 +0,0 @@
----
```
Comment (paweldomas) [inline .github/ISSUE_TEMPLATE/bug_report.md:1]: this file can probably stay as well
